docs/development/release_management.rst

```diff
@@ -310,28 +318,36 @@ Release Process
 #. Update the upgrade testing boxes following this process:
    :ref:`updating_upgrade_boxes`.
 
-
 Post-Release
 ------------
 
-Now it's time to backport the changelog from the release branch into the ``develop`` branch and bump
-the SecureDrop version so that it's ready for the next round of QA testing.
-
-We backport the changelog by cherry-picking any commits that modified ``changelog.md`` during the
-release. You can look at the file history by checking out the release branch and running:
-``git log --pretty=oneline changelog.md``. The output will contain the commit hashes associated with
-the release. Create a new branch based on ``develop`` and cherry-pick these commits using the
-``-x`` flag.
-
-Now you're ready to bump the SecureDrop version on your new branch. There are a bunch of version
-files that'll need to be updated in order to set up the upgrade test for the next release. We do
-this by running the version-updater script and specifying the new version number, which will be the
-next minor version with ``~rc1`` appended. For example, if the release is 1.3.0, then you'll run:
-``securedrop/bin/dev-shell ../update_version.sh 1.4.0~rc1``  (``dev-shell`` is a script that starts
-a container so that we can ensure ``dch`` is installed). Accept all the default changes from the
-``update_version.sh`` script. You'll only need to add your commit message. Once you're done, sign
-your commit and make a PR to merge these changes into ``develop``.
-
-The only thing left to do is to monitor the `FPF support portal <https://support.freedom.press>`_
-and the `SecureDrop community support forum <https://forum.securedrop.org/c/support>`_ for any new
-user issues related to the release.
+1. Backport the changelog from the release branch into ``develop``.
+
+   a. Collect the hashes of all the commits that modified ``changelog.md`` during the release::
+
+         git log --pretty=oneline changelog.md
+
+   #. From a new branch based on ``develop``, cherry-pick each commit in the ``git log`` output
+      from the previous step. Make sure to use the ``-x`` flag so that the original commit is
+      appended to the new commit.
+
+#. Bump the SecureDrop version so that it's ready for the next release.
+
+   a. Create a new minor release candidate. Only add a commit message and accept the default changes
+      for everything else (it's fine to leave the changelog entries with empty bullets). For
+      example, if the release is 1.3.0, then you'll run::
+
+         ./update_version.sh 1.4.0~rc1
```

Comment:
As a side note, when `dch` is not installed, the script helpfully prints:

```
./update_version.sh 1.8.0~rc1
dch is missing
run with securedrop/bin/dev-shell ../update_version.sh <version>
```

So I think this is a fine simplification, and folks who need the container can still find out how to use it that way.